api/v1/helpers/verbs/utils.js

```diff
@@ -182,18 +182,27 @@ function isWritable(req, modelInst) {
       jwtUtil.getTokenDetailsFromRequest(req)
       .then((resObj) => modelInst.isWritableBy(resObj.username))
       .then((ok) => ok ? resolve(modelInst) :
-          reject(new apiErrors.ForbiddenError())
+        reject(new apiErrors.ForbiddenError(
+          'Resource not writable for provided token'))
       )
-      .catch((err) => reject(err));
+      .catch(reject);
     } else if (req.user) {
       // try to use the logged-in user
       modelInst.isWritableBy(req.user.name)
       .then((ok) => ok ? resolve(modelInst) :
-          reject(new apiErrors.ForbiddenError())
+        reject(new apiErrors.ForbiddenError(
+          'Resource not writable by this user'))
       )
-      .catch((err) => reject(err));
+      .catch(reject);
     } else {
-      reject(new apiErrors.ForbiddenError());
+      // check if isWritable with no user
+      // when not passed a user, isWritable will return true if
+      // the resource is not write protected, false if it is
```

Comment:
I think we want to use 
`/*
*
*/`  for multi-line comments instead of `//`. We can talk care of this later.